test/format.spec.js

```diff
@@ -0,0 +1,104 @@
+import { expect } from "chai";
+import { format } from "../src/format";
+import { notInGivenKeys, transformToStrs } from "./helper";
+import romanizedExpectedKeys from "./layouts/romanized/helpers/expected_keys";
+import romanizedTestKeys from "./layouts/romanized/helpers/test_keys";
+import traditionalExpectedKeys from "./layouts/traditional/helpers/expected_keys";
+import traditionalTestKeys from "./layouts/traditional/helpers/test_keys";
+
+describe("format", () => {
+  describe("format(..., option)", () => {
+    it("should not throw", () => {
+      const layouts = ["romanized", "traditional"];
+
+      expect(() => format("trtr")).to.not.throw();
+      expect(() =>
+        layouts.forEach((layout) => format("trtr", { layout: layout }))
+      ).to.not.throw();
+    });
+
+    it("should throw", () => {
+      expect(() => format("trtr", { layout: "not a layout" })).to.throw();
+    });
+  });
+
+  describe("format(..., romanized)", () => {
+    const romanizedOption = { layout: "romanized" };
+
+    it("should return mapped romanized string", () => {
+      const inputStrs = transformToStrs(romanizedTestKeys);
```

Comment:
for these tests, I think it would be better if we tested well formed text against a nepali string (and not use helpers created to test another module). 
So, I'd say it's better if we tested upto 5 different sentences possibly covering all kinds of edge cases, right here in this code block. 5 asserts should be enough for now.

- simple bunch of words
- words that have combined forms (like `श्री.....` some examples here https://github.com/rhoit/m17n-ne#223-phonetic-completion)
- words that make use of `ZWJ` and `ZWNJ`
- other odds combinations